src/HTTPure/Streamable.purs

```diff
@@ -0,0 +1,19 @@
+module HTTPure.Streamable
+  ( class Streamable
+  , toStream
+  ) where
+
+import Node.Buffer as Buffer
+import Node.Stream as Stream
+
+foreign import createStreamFromString :: String -> Stream.Readable ()
+foreign import createStreamFromBuffer :: Buffer.Buffer -> Stream.Readable ()
+
+class Streamable s where
+  toStream :: s -> Stream.Readable ()
+
+instance streamableString :: Streamable String where
+  toStream = createStreamFromString
+
+instance streamableBuffer :: Streamable Buffer.Buffer where
+  toStream = createStreamFromBuffer
```

Comment:
@hdgarrood @paf31 I see you guys are the major committers to https://github.com/purescript-node/purescript-node-streams... This module feels like something that belongs there but it's not really a direct mapping to Node APIs... what do you guys think?  If you want it, I'm happy to submit a PR to put it there

Reply:
That’s not the sort of thing I’d want to put in the node bindings - as you say, it doesn’t really correspond to any part of the node api, and also lawless typeclasses are something I’d want to avoid if possible anyway.